src/components/NoteCard.tsx

```diff
@@ -71,6 +71,16 @@ export default function NoteCard({
               {note.created_by?.name && (
                 <ProfileCard profile={note.created_by} />
               )}
+
+              {note.is_owner && (
+              <Button mode="bleed" tone="secondary" size="xs" asChild>
+                    <Link href={paths.editNote(note.handle)}>
+                      <EditIcon className="w-[1.25em]" />
+                      Editar
+                    </Link>
+              </Button>
+              )}
+
```

Comment:
I think we need to redesign the notes card to better accommodate for these actions... In the meantime, can you move it after the share button, make both `size="sm"` and apply the `w-[1.25em]` class to the share icon?